tests/WatcherCommandTest.php

```diff
@@ -2,12 +2,14 @@
 
 namespace Spatie\PhpUnitWatcher\Test;
 
+use PHPUnit\Framework\Attributes\Test;
 use PHPUnit\Framework\TestCase;
 use Spatie\PhpUnitWatcher\WatcherCommand;
 
 class WatcherCommandTest extends TestCase
 {
     /** @test */
```

Comment:
These docblocks can be removed, now that we use the annotation

Reply:
I left them because I saw an example where both were kept for older PHP unit versions, I think. I removed the doc blocks in favor of annotations.